pkg/config/config.go

```diff
@@ -1999,6 +2025,30 @@ func (r *RuntimeHandler) validateRuntimeExecCPUAffinity() error {
 	return fmt.Errorf("invalid exec_cpu_affinity %q", r.ExecCPUAffinity)
 }
 
+// validateRuntimeSeccompProfile tries to load the RuntimeHandler seccomp profile.
+func (r *RuntimeHandler) validateRuntimeSeccompProfile() error {
+	if r.SeccompProfile == "" {
+		r.seccompConfig = nil
+	} else {
```

Comment:
nit: it can early-return and can reduce a nesting level.
```suggestion
	if r.SeccompProfile == "" {
		r.seccompConfig = nil
		return nil
	}
```

Reply:
Done. However, our linters insist adding an empty line before return statement which isn't ideal in this case.